api/libretime_api/storage/models/file.py

```diff
@@ -28,7 +28,14 @@ class ImportStatus(models.IntegerChoices):
     is_hidden = models.BooleanField(db_column="hidden", blank=True, null=True)
 
     is_accessed = models.IntegerField(db_column="currentlyaccessing")
-    edited_by = models.ForeignKey(
+    is_scheduled = models.BooleanField(db_column="is_scheduled", blank=True, null=True)
+    is_part_of = models.BooleanField(db_column="is_playlist", blank=True, null=True)
+
+    uploaded_at = models.DateTimeField(db_column="utime", blank=True, null=True)
```

Comment:
maybe on instead of at?

Reply:
Maybe, I try to follow a convention and I am used to `created_at`, `modified_at`. I'll check if I already started to use `_at` somewhere else.

Reply:
I don't really mind either way, but I agree that keeping to a standard convention is important

Reply:
https://stackoverflow.com/questions/15038726/ruby-on-rails-convention-for-naming-date-time-fields

Reply:
I'm happy with that